src/sentry/static/sentry/app/views/settings/components/forms/controls/radioGroup.tsx

```diff
@@ -0,0 +1,126 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+import isPropValid from '@emotion/is-prop-valid';
+
+import {growIn} from 'app/styles/animations';
+
+type Props = {
+  value: string | number | null;
+
+  // An array of [id, name, description]
+  choices: [string, string, string?][];
+  disabled?: boolean;
+  label: string;
+  onChange: (id: string, e: React.MouseEvent) => void;
+};
+
+const RadioGroup: React.FC<Props> = ({
+  value,
+  disabled,
+  choices,
+  label,
+  onChange,
+  ...props
+}) => {
+  return (
+    <div {...props} role="radiogroup" aria-labelledby={label}>
+      {(choices || []).map(([id, name, description], index) => {
+        const isSelected = value === id;
+
+        return (
+          <RadioLineItem
+            key={index}
+            onClick={e => !disabled && onChange(id, e)}
+            role="radio"
+            index={index}
+            aria-checked={isSelected}
+            disabled={disabled}
+          >
+            <RadioLineButton type="button" disabled={disabled}>
+              {isSelected && (
+                <RadioLineButtonFill disabled={disabled} animate={value !== ''} />
+              )}
+            </RadioLineButton>
+            <RadioLineText disabled={disabled}>{name}</RadioLineText>
+            {description && (
+              <React.Fragment>
+                <div />
```

Comment:
Would it make sense to set a `margin` on `Description` instead? Otherwise, you should add a comment explaining why this `div` is needed.

Reply:
It's there because of css grid - I rather rely on the grid than an explicit margin in case the button size changes. (Added a comment)

Reply:
👍